docs/API-Reference/bpmn-js/importXML.md

```diff
@@ -10,3 +10,62 @@ grand_parent: "API Reference"
 ---
 
 # importXML
+
+`importXML` API parses and renders a BPMN 2.0 diagram.
+
+## Usage
+
+importXML(xml: _[String](#)_, bpmnDiagram: _[ModdleElement](#)_ or _[String](#)_): Promise<_[ImportXMLResult](#importxmlresult)_, _[ImportXMLError](#importxmlerror)_>
```

Comment:
_[String](#)_ doesn't require a link IMO, so it can be just _`String`_

Reply:
Ah yeah good idea. Updated the empty links in input parameters.

Reply:
Same goes for the ModdleElement parameter since it does not point to any page.